dl.py

```diff
@@ -1,13 +1,36 @@
 import sys
 from multiprocessing import Process
 from pytube import YouTube
+import ffmpeg
 
 
 def dl_vid(url):
     yt = YouTube(url)
-    print('downloading: ' + yt.title)
-    yt.streams.filter(progressive=True, file_extension='mp4').order_by('resolution').desc().first().download() 
-    print('finished: ' + yt.title)
+    title = yt.title
+    print('Downloading: ' + title)
+
+    # Check if text contains # symbol, if so replace as ffmpeg doesn't like #
+    if title.__contains__("#"):
```

Comment:
thoughts on using `in` rather than  `__contains__` ? it feels more python-y to me